test/integration/__snapshots__/lerna-publish.test.js.snap

```diff
@@ -8,7 +8,7 @@ exports[`commit: updates fixed versions 1`] = `
 exports[`commit: updates independent versions 1`] = `
 "Publish
 
- - package-1@2.0.0
+ - package-1@__TEST_VERSION__
```

Comment:
This technically passes, but is incorrect. I need to fix this upstream. In the mean time, I will parse the test failures and merge stuff that passes everything else.

You'll want to remove the changes to this file so your PR doesn't conflict with the fixes I make in `master`.